uberduck_ml_dev/models/rvc/rvc.py

```diff
@@ -106,6 +106,51 @@ def forward(self, phone, pitch, lengths):
         x = self.proj(x) * x_mask
         return x, x_mask
 
+class TextEncoder768(nn.Module):
```

Comment:
can we just pass another argument that controls 256 v 768?  not totally necessary but worth considering